calculator/src/App.js

```diff
@@ -0,0 +1,83 @@
+/* eslint no-eval: 0 */
+import React, { Component } from 'react';
+import './App.css';
+import Results from './components/Results';
+import Buttons from "./components/Buttons";
+
+class App extends Component {
+    constructor(){
+        super();
+
+        this.state = {
+            result: ""
```

Comment:
A thought process around avoiding using eval is to keep track of your operator in the state here